docs/en/operation/User-CLI.md

```diff
@@ -823,15 +825,18 @@ Later jobs must wait until some earlier jobs to finish. The default value is `30
 A lower value means slower execution but also being nicer to the other users of the job service.
 * `--overwrite`: Whether to overwrite the destination. Default is true.
 * `--batch-size`: Specifies how many files to be batched into one request. The default value is `20`. Notice that if some task failed in the batched job, the whole batched job would fail with some completed tasks and some failed tasks.
-* `--wait`: true or false specifies whether to wait for command execution to finish. If not explicitly shown then default to wait (true).
+* `--not-wait`: Specifies whether to wait for command execution to finish. If not explicitly shown then default to wait or run synchronously.
 ```console
 $ ./bin/alluxio fs distributedCp --active-jobs 2000 /data/1023 /data/1024
 ```
 
 ### distributedLoad
 
 The `distributedLoad` command loads a file or directory from the under storage system into Alluxio storage distributed
-across workers using the job service. The job is a no-op if the file is already loaded into Alluxio. User will get a job control ID after the command submission is successful in async mode. In sync mode, user will get a job control Id after the command finishes.
+across workers using the job service. The job is a no-op if the file is already loaded into Alluxio.
+User will get a job control ID after the command submission is successful in async mode.
```

Comment:
but sync mode is default? why explain async mode first?

Reply:
swapped the sequence and added more details